.github/workflows/nightly_tests.yml

```diff
@@ -240,10 +240,80 @@ jobs:
           junit-report-base-name: clang18_test_report
           retention-days: 2
 
+  clang18_build_oldest_supported_dependencies:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/4c-multiphysics/4c-dependencies-ubuntu24.04-oldest-supported:8115e93e
+      options: --user root --env OMPI_ALLOW_RUN_AS_ROOT=1 --env OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
+    defaults:
+      run:
+        shell: bash
+    steps:
```

Comment:
You forgot 
```
outputs:
test-chunks: ${{ steps.set-matrix.outputs.chunk-array }}
```
That's why the nightly pipeline fails https://github.com/4C-multiphysics/4C/actions/runs/15220171403
I will push a fix.